From d3481ee7b7d104ef53ead4d35b9a9254c64bb87a Mon Sep 17 00:00:00 2001
From: Sam James <sam@gentoo.org>
Date: Sun, 12 Jun 2022 20:49:09 +0100
Subject: [PATCH] Fix build with GCC 12 (missing <time.h> include)
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fixes build failure with GCC 12:
```
Toolbar.cc: In member function ‘void Toolbar::reconfigure()’:
Toolbar.cc:260:17: error: ‘time’ was not declared in this scope; did you mean ‘Time’?
  260 |   time_t ttmp = time(NULL);
      |                 ^~~~
      |                 Time
Toolbar.cc:265:21: error: ‘localtime’ was not declared in this scope; did you mean ‘clock_timer’?
  265 |     struct tm *tt = localtime(&ttmp);
      |                     ^~~~~~~~~
      |                     clock_timer
```

Bug: https://bugs.gentoo.org/851603
---
 src/Toolbar.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Toolbar.cc b/src/Toolbar.cc
index e88b8f6b..8ccad201 100644
--- a/src/Toolbar.cc
+++ b/src/Toolbar.cc
@@ -38,6 +38,7 @@
 #include <X11/Xutil.h>
 #include <sys/time.h>
 #include <assert.h>
+#include <time.h>
 
 
 long nextTimeout(int resolution)
